### ISSUE TITLE
Check compatibility with Angular 7.

### ISSUE THREAD

**Issue by earshinov** [madoar/angular-archwizard#163]
[Angular 7.0 is released](https://blog.angular.io/version-7-of-angular-cli-prompts-virtual-scroll-drag-and-drop-and-more-c594e22e7b8c). If angular-archwizard is compatible with this version:
- this fact should be documented in the README;
- dependency versions in `package.json` should be updated.

**Comment by earshinov** [comment]
I tried to checkout [angular-archwizard-demo](https://github.com/madoar/angular-archwizard-demo/) repo and upgrade it to Angular 7 (*). Everything went fine and the project compiled without a single error both in development (`ng serve`) and production modes (`ng build --prod`). Based on that, I think that angular-archwizard can safely declare compatibility with Angular 7.

(*) In case someone needs them, here are changes one has to make in `angular-archwizard-demo/package.json` to upgrade the project to Angular 7:

```diff
diff --git a/package.json b/package.json
index 1c27e4f..f28965a 100644
--- a/package.json
+++ b/package.json
@@ -12,29 +12,29 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.1.9",
-    "@angular/common": "^6.1.9",
-    "@angular/compiler": "^6.1.9",
-    "@angular/core": "^6.1.9",
-    "@angular/forms": "^6.1.9",
-    "@angular/http": "^6.1.9",
-    "@angular/platform-browser": "^6.1.9",
-    "@angular/platform-browser-dynamic": "^6.1.9",
-    "@angular/router": "^6.1.9",
+    "@angular/animations": "^7.0.0",
+    "@angular/common": "^7.0.0",
+    "@angular/compiler": "^7.0.0",
+    "@angular/core": "^7.0.0",
+    "@angular/forms": "^7.0.0",
+    "@angular/http": "^7.0.0",
+    "@angular/platform-browser": "^7.0.0",
+    "@angular/platform-browser-dynamic": "^7.0.0",
+    "@angular/router": "^7.0.0",
+    "angular-archwizard": "^4.0.0",
     "bootstrap": "^4.0.0-alpha.6",
     "core-js": "^2.4.1",
     "font-awesome": "^4.7.0",
     "jquery": "^3.2.1",
-    "angular-archwizard": "^4.0.0",
     "rxjs": "^6.3.3",
     "tether": "^1.4.0",
     "zone.js": "^0.8.4"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.8.0",
-    "@angular/cli": "^6.2.3",
-    "@angular/compiler-cli": "^6.1.9",
-    "@angular/language-service": "^6.1.9",
+    "@angular/cli": "^7.0.0",
+    "@angular/compiler-cli": "^7.0.0",
+    "@angular/language-service": "^7.0.0",
     "@types/jasmine": "^2.8.6",
     "@types/node": "^9.4.7",
     "codelyzer": "^4.2.1",
@@ -49,6 +49,6 @@
     "protractor": "^5.3.0",
     "ts-node": "^5.0.1",
     "tslint": "^5.9.1",
-    "typescript": "2.9.2"
+    "typescript": "3.1.3"
   }
 }
```

**Comment by earshinov** [comment]
Created PR https://github.com/madoar/angular-archwizard/pull/165 with changes in `package.json`. The README file does not need to be updated.

**Comment by madoar** [comment]
II've just released `angular-archwizard` version `4.0.0` to [npm](https://www.npmjs.com/package/angular-archwizard)! `angular-archwizard` version `4.0.0` uses Angular 7. Feel free to give it a try!
